src/WebJobs.Script/Binding/ExtensionBinding.cs

```diff
@@ -45,87 +44,30 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes(Type para
 
         public override async Task BindAsync(BindingContext context)
         {
-            // All the below BindAsync logic is temporary IBinder support
-            // Once the Invoker work is done, we'll be binding directly
-            Collection<Attribute> resolvedAttributes = ResolveAttributes(_attributes, context.BindingData);
-            var attribute = resolvedAttributes.First();
-            var additionalAttributes = resolvedAttributes.Skip(1).ToArray();
-            RuntimeBindingContext runtimeContext = new RuntimeBindingContext(attribute, additionalAttributes);
-
-            // TEMP: We'll be doing away with this IBinder code
+            context.Attributes = _attributes.ToArray();
+
             if (_binding.DefaultType == typeof(IAsyncCollector<byte[]>))
             {
-                await BindAsyncCollectorAsync<byte[]>(context, runtimeContext);
+                await BindAsyncCollectorAsync<byte[]>(context);
             }
             else if (_binding.DefaultType == typeof(IAsyncCollector<JObject>))
             {
-                await BindAsyncCollectorAsync<JObject>(context, runtimeContext);
+                await BindAsyncCollectorAsync<JObject>(context);
             }
             else if (_binding.DefaultType == typeof(Stream))
             {
-                await BindStreamAsync(context, Access, runtimeContext);
+                await BindStreamAsync(context, Access);
             }
             else if (_binding.DefaultType == typeof(JObject))
             {
-                var result = await context.Binder.BindAsync<JObject>(runtimeContext);
+                var result = await context.Binder.BindAsync<JObject>(_attributes.ToArray());
                 if (Access == FileAccess.Read)
                 {
                     context.Value = result;
                 }
             }
         }
 
-        // TEMP - Since we're still using IBinder for non C#, we have to construct the Attributes
-        // This code will go away soon
-        private Collection<Attribute> ResolveAttributes(Collection<Attribute> attributes, IReadOnlyDictionary<string, string> bindingData)
```

Comment:
This was all the logic that was previously resolving binding parameters in attributes (e.g. the `{id}` in `[Queue("input/{id}")]`). We now let the bindings do that in the SDK naturally.